scripts/vulnxscan/README.md

```diff
@@ -6,154 +6,233 @@ SPDX-License-Identifier: Apache-2.0
+* [Supported scanners](#supported-scanners)
+   * [Nix and OSV vulnerability database](#nix-and-osv-vulnerability-database)
+   * [Nix and Grype](#nix-and-grype)
+   * [Vulnix](#vulnix)
+* [Vulnxscan usage examples](#vulnxscan-usage-examples)
+   * [Running vulnxscan as flake](#running-vulnxscan-as-flake)
+   * [Find vulnerabilities impacting runtime dependencies](#find-vulnerabilities-impacting-runtime-dependencies)
+   * [Find vulnerabilities given SBOM as input](#find-vulnerabilities-given-sbom-as-input)
+   * [Find vulnerabilities impacting buildtime and runtime dependencies](#find-vulnerabilities-impacting-buildtime-and-runtime-dependencies)
+* [Footnotes and future work](#footnotes-and-future-work)
+
 
 ## Getting Started
 To get started, follow the [Getting Started](../../README.md#getting-started) section from the main [README](../../README.md).
 
 ## Usage Examples
-In the below examples, we use nix package `git` as an example target.
-To install nix `git` package and print out its out-path on your local system, try something like:
+In the below examples, we use `sbomnix` itself as an example target for `vulnxscan`.
+To get the target out-path, build `sbomnix` with `nix-build`:
 ```bash
-nix-shell -p git --run exit && nix eval -f '<nixpkgs>' 'git.outPath'
-```
-Which outputs the example target out-path on your local system:
+$ git clone https://github.com/tiiuae/sbomnix
+$ cd sbomnix
+$ nix-build
 ```
-==> out-path: /nix/store/5207hz4f779nz4z62zaa5gdjqzqz1l4g-git-2.39.0
+Which creates a `result` symlink that points to `sbomnix` out-path on your local system:
+```bash
+$ ls -l result
+lrwxrwxrwx   result -> /nix/store/6y93c1b4453mjqn6ar6sszcf06svr3jl-python3.10-sbomnix-1.2.0
 ```
 
-#### Example: finding OSV vulnerabilities for nix target
-This example demonstrates using `osv.py` to manually find vulnerabilities for nix target (derivation or out path).
+## Supported scanners
+### Nix and OSV vulnerability database
+[OSV](https://osv.dev/) is a vulnerability database for open source projects [initiated by Google](https://security.googleblog.com/2021/02/launching-osv-better-vulnerability.html). 
+
+[OSV database](https://osv.dev/list?ecosystem=) currently [does not support nix ecosystem](https://ossf.github.io/osv-schema/#affectedpackage-field), so queries that specify nix as ecosystem would not return any matches. For this reason `vulnxscan` currently does not use Google's official [OSV-Scanner](https://security.googleblog.com/2022/12/announcing-osv-scanner-vulnerability.html) front-end, but implements it's own OSV client demo in [osv.py](./osv.py).
+
+`osv.py` sends queries to [OSV API](https://osv.dev/docs/) without specifying the ecosystem, only the target package name and version. At the time of writing, such queries to OSV API return vulnerabilities that match the given package and version across all ecosystems. As a result, the OSV vulnerabilities for nix ecosystem will include false positives. 
```

Comment:
Is 'nix ecosystem' some special name? Like a variable. If not, we should write Nix as you write Google, not google.

Reply:
It's not.

Fixed in https://github.com/tiiuae/sbomnix/commit/b424d9b917e0de3151289f417ef927f3726e339a